docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -501,6 +501,14 @@ of startup on its own.
 Tune the container's pids limit.
 This is equivalent to the Podman `--pids-limit` option.
 
+### `Pod=`
+
+Specify a pod to link the container to. This has the same format as the `--pod` option
```

Comment:
I assume that other users may fall into the same trap as I did, so I propose that `Pod=` should only accept `.pod` units as input.

I don't think we should allow for mixing containers that run inside of a systemd unit with a pod that runs outside of systemd.

Reply:
Some replies. But, if you insist, I'm fine with forcing it. Just need to get your answer on the first question

> It's trap I expect people to fall into. Without requiring the value to point to a `.pod` unit, there are no dependencies.

If we mandate that the pod name must link to a Quadlet pod, do we need to force that it contains the `.pod` suffix?

> I am very much in favor in having those discussions. At least, I cannot imagine a reasonable use case where a container should be running under systemd but not the `.pod` (infra container).

We didn't think people will care about the network name either

> So turning it the other way around, without the requirement we can expect issues with "Why are my containers not started when I start the `.pod`".

One can also ask "why didn't the volume get created automatically when I started my `.container`?"

Reply:
> Should Quadlet still force that the suffix is set. I.e, assuming `test.pod` exists, can the users set `test` or must they set `test.pod`

Do we have a precedent in Quadlet for that?  I think that requiring the suffix may be less ambiguous.

Reply:
I've change the code to enforce `.pod`. The tests cover failing if the value is not `.pod` or if the `.pod` file does not exist

Reply:
Thanks a lot, @ygalblum !